examples/postcards_template.rb

```diff
@@ -61,32 +61,38 @@
 
 # create a to address
 to_address = lob.addresses.create(
-  name: "ToAddress",
-  address_line1: "120 6th Ave",
-  address_city: "Boston",
-  address_state: "MA",
-  address_country: "US",
-  address_zip: 12345
+  {
+    "name": "ToAddress",
+    "address_line1": "120 6th Ave",
+    "address_city": "Boston",
+    "address_state": "MA",
+    "address_country": "US",
+    "address_zip": 12345
+  }
 )
 
 # create a from address
 from_address = lob.addresses.create(
-  name: "FromAddress",
-  address_line1: "120 6th Ave",
-  address_city: "Boston",
-  address_state: "MA",
-  address_country: "US",
-  address_zip: 12345
+  {
+    "name": "FromAddress",
+    "address_line1": "120 6th Ave",
+    "address_city": "Boston",
+    "address_state": "MA",
+    "address_country": "US",
+    "address_zip": 12345
+  }
 )
 
 # send a postcard
 pp lob.postcards.create(
```

Comment:
You'll want to remove this debugging code.

Reply:
If you're referring to the usage of `pp`, that one is actually a replacement for `puts`, which was yielding pretty unreadable output. It's not for debugging purposes.